pkg/exporter/sumologicexporter/README.md

```diff
@@ -114,6 +114,9 @@ exporters:
       # num_seconds is the number of seconds to buffer in case of a backend outage,
       # requests_per_second is the average number of requests per seconds.
       queue_size: <queue_size>
+
+    # default = false
+    sticky_session_enabled: {true, false}
```

Comment:
It would be nice to have some description for customers.

I would prefer name without `_enabled` but not sure if it's better or not :sweat_smile: